node/node.go

```diff
@@ -59,12 +59,16 @@ type Node struct {
 
 	DASer *das.DASer `optional:"true"`
 
+	// PluginOutlet serves as an arbitrary type for plugins to return so that fx
+	// will call them
+	PluginOutlet
```

Comment:
Plugins have to have a service that initializes this `PluginOutlet` types so that fx will call them at all.